tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -160,6 +160,15 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 
 			helper.CmdShouldPass("odo", "service", "delete", svcFullName, "-f")
 		})
+
+		It("should fail to create operand if a non existing param is provided", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs")
+			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
+			stdErr := helper.CmdShouldFail("odo", "service", "create", fmt.Sprintf("%s/EtcdCluster", etcdOperator), "-p", "sze=4", "--project", commonVar.Project)
```

Comment:
You could use a param like 'unknown-param' instead of 'sze' as the typo could be difficult to catch for some reader

Reply:
ack, good suggestion. thank you.